student.rb

```diff
@@ -1,12 +1,19 @@
-require './person.rb'
+require './person'
 
 class Student < Person
-  def initialize(age, classroom, name = "Unknown", parent_permission = true)
+  attr_reader :classroom
+
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
   end
 
   def play_hooky
     "¯\(ツ)/¯"
   end
+
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self)
```

Comment:
- [ ] There should be a condition here to check whether the students exist already to avoid duplication, kindly work on that.